datagrid/datagrid65-basic.json

```diff
@@ -286,9 +286,14 @@
                                         "protocol": "TCP"
                                     },
                                     {
-                                        "name": "hotrod",
+                                        "name": "hotrod-internal",
                                         "containerPort": 11222,
                                         "protocol": "TCP"
+                                    },
+                                    {
+                                        "name": "hotrod-external",
```

Comment:
I think this is better named as simply hotrod, since that is the port that external clients should use for accessing hotrod.  I think it's sufficient that the internal port is named hotrod-internal.  Notice that the service definition uses port 11333.